File: Revisão Prática/Comando de decição/cSimpleCommandProgram.c
// Comandos de Decisão em C
#include <stdio.h>

int main() {
    int num1;
    
    printf("Digite um número inteiro (positivo, negativo ou zero): ");
    
    if (scanf("%d", &num1) != 1) {
        printf("Erro: Entrada inválida.\n");
        return 1;
    }
    
    printf("\n--- Classificação (C) ---\n");
    
    // Comandos de Decisão: if, else if, else
    if (num1 > 0) {
        printf("O número %d é POSITIVO.\n", num1);
    } else if (num1 < 0) {
        printf("O número %d é NEGATIVO.\n", num1);
    } else {
        printf("O número %d é ZERO.\n", num1);
    }
    
    return 0;
}

File: Revisão Prática/Comando de decição/cSimpleCommandProgram.java
// Comandos de Decisão em Java
import java.util.Scanner;

public class ComandosDecisao {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite um número inteiro (positivo, negativo ou zero): ");
        
        if (scanner.hasNextInt()) {
            int num1 = scanner.nextInt();
            
            System.out.println("\n--- Classificação (Java) ---");
            
            // Comandos de Decisão: if, else if, else
            if (num1 > 0) {
                System.out.println("O número " + num1 + " é POSITIVO.");
            } else if (num1 < 0) {
                System.out.println("O número " + num1 + " é NEGATIVO.");
            } else {
                System.out.println("O número " + num1 + " é ZERO.");
            }
        } else {
            System.out.println("Erro: Entrada inválida.");
        }
        scanner.close();
    }
}